src/app/pages/about-us/bios.js

```diff
@@ -127,266 +127,238 @@ let bios = {
         {
             name: 'Richard Baraniuk',
             title: 'Director',
-            bio: `Richard G. Baraniuk is the Victor E. Cameron Professor of
-            Electrical and Computer Engineering at Rice University and the
-            founder and director of OpenStax. In 1999, Dr. Baraniuk launched
-            Connexions (now OpenStax CNX), one of the world’s first and today one
-            of the world’s largest “open education” platforms, providing free and
-            remix able e-textbooks to millions of users from 200 countries. More
-            recently, through his research in machine learning, he is enhancing
-            the OpenStax textbooks with personalized learning features that
-            optimize each student’s learning experience based on each their
-            background, context, and learning goals. For his education projects,
-            Dr. Baraniuk has received the Eta Kappa Nu C. Holmes MacDonald
-            National Outstanding Teaching Award, the Tech Museum Laureate Award,
-            the Internet Pioneer Award from the Berkman Center for Internet and
-            Society at Harvard Law School, the World Technology Network Education
-            Award, the IEEE Signal Processing Society Education Award, and has
-            been named one of Edutopia Magazine’s Daring Dozen Education Innovators.
-            For his research projects in machine learning, he has received numerous
-            national awards and has been elected a Fellow of IEEE and AAAS.`
+            bio: `Richard Baraniuk is the Victor E. Cameron Professor of Electrical
+            and Computer Engineering at Rice University and the founder/director of
+            OpenStax. In 1999, Dr. Baraniuk launched Connexions (now OpenStax CNX),
+            one of the world’s first and largest “open education” platforms. He now works
+            on enhancing OpenStax textbooks with personalized learning features through
+            his research in machine learning.`,
+            // image: 'richard_baraniuk.jpg',
+            bgColor: 'gray',
+            textColor: 'orange'
```

Comment:
Fields have been removed

On 04/28/2016 05:28 AM, Roy Johnson wrote:
>
> In src/app/pages/about-us/bios.js 
> <https://github.com/openstax/os-webview/pull/290#discussion_r61405456>:
>
> > -            National Outstanding Teaching Award, the Tech Museum Laureate Award,
> > -            the Internet Pioneer Award from the Berkman Center for Internet and
> > -            Society at Harvard Law School, the World Technology Network Education
> > -            Award, the IEEE Signal Processing Society Education Award, and has
> > -            been named one of Edutopia Magazine’s Daring Dozen Education Innovators.
> > -            For his research projects in machine learning, he has received numerous
> > -            national awards and has been elected a Fellow of IEEE and AAAS.`
> > +            bio: `Richard Baraniuk is the Victor E. Cameron Professor of Electrical
> > +            and Computer Engineering at Rice University and the founder/director of
> > +            OpenStax. In 1999, Dr. Baraniuk launched Connexions (now OpenStax CNX),
> > +            one of the world’s first and largest “open education” platforms. He now works
> > +            on enhancing OpenStax textbooks with personalized learning features through
> > +            his research in machine learning.`,
> > +            // image: 'richard_baraniuk.jpg',
> > +            bgColor: 'gray',
> > +            textColor: 'orange'
>
> No. Colors are assigned by the program now.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub 
> <https://github.com/openstax/os-webview/pull/290/files/c1984d9c5b0f0b0ff4e35983fcd7e69a85745d41#r61405456>
>
> !DSPAM:1106,5721e5cb276741639411586!